Pyrado/pyrado/algorithms/base.py

```diff
@@ -434,3 +436,20 @@ def reached_checkpoint(self, meta_info: dict = None):
         self._curr_checkpoint = next % (self._num_checkpoints + 1) if next > 0 else next  # no modulo for negative count
 
         self.save_snapshot(meta_info)
+
+
+class ExposedSampler:
```

Comment:
Please put this class in a separate file in .../algorithms

Reply:
dd47b5a3b84e9a016e6eace74fd10286cedd440d